src/com/palmergames/bukkit/towny/listeners/TownyEntityListener.java

```diff
@@ -243,6 +228,24 @@ public void onEntityDeath(EntityDeathEvent event) {
 		}
 		
 	}
+
+
+	@EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
+	public void onEntityTargetLivingEntity(EntityTargetLivingEntityEvent event) {
+		if (plugin.isError()) {
+			return;
+		}
+
+		if (event.getTarget() instanceof Player) {
+
+			Player target = (Player)event.getTarget();
+			if (event.getReason().equals(EntityTargetEvent.TargetReason.TEMPT)) {
+				if (!PlayerCacheUtil.getCachePermission(target, event.getEntity().getLocation(), target.getInventory().getItemInMainHand().getType(), TownyPermission.ActionType.ITEM_USE)) {

```

Comment:
Rather than relying on all the foods to be in the item_use list (which would make it so outsiders could not feed themselves,) would it not make more sense to try a break test? I believe we check if player's are able to break dirt or grass blocks when we're testing if they can kill animals.

Reply:
If you change it to this, I can get behind this PR.
https://github.com/TownyAdvanced/Towny/blob/master/src/com/palmergames/bukkit/towny/utils/CombatUtil.java#L196